solidity/scripts/rewards.js

```diff
@@ -40,74 +40,87 @@ function formatAmount(amount, decimals) {
 }
   
 module.exports = async function() {
-  const accounts = await getAccounts();
-  const token = await KeepToken.deployed();
-  const tokenStaking = await TokenStaking.deployed();
-  const contractService = await KeepRandomBeaconService.deployed();
-  const keepRandomBeaconService = await KeepRandomBeaconServiceImpl.at(contractService.address);
-  const keepRandomBeaconOerator = await Operator.deployed();
-
-  const magpie = accounts[5];
-  const requestor = accounts[5];
-  const owner = accounts[0];
+  try {
+
+    const accounts = await getAccounts();
+    const token = await KeepToken.deployed();
+    const tokenStaking = await TokenStaking.deployed();
+    const contractService = await KeepRandomBeaconService.deployed();
+    const keepRandomBeaconService = await KeepRandomBeaconServiceImpl.at(contractService.address);
+    const keepRandomBeaconOerator = await Operator.deployed();
```

Comment:
Typo `keepRandomBeaconOerator`. Also, please rename `Operator` to the full contract name.